man/systemctl.xml

```diff
@@ -1923,6 +1932,17 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
       or
       <programlisting># systemctl link /path/to/foo.service</programlisting>
       </para>
+
+      <para>For the use of operating multiple units, please note that the units are operated sequentially instead of
+      operated in a single transaction. For example, we have 2 service units: <filename>first.service</filename> and
```

Comment:
Thanks for the review. English is not my native language either, so don't be sorry for anything. However, I saw 2 issues in your statement:

1. it's "more than one unit" instead of "more than two units"
2. `systemctl start first second` will make the ordering dependency take effect. Therefore, I don't think it's safe to say "no ordering dependency between them are considered in the operation".

Reply:
1. Oops...
2. Is it? I do not check the relevant code. But if so, what is the real limitation here?

Reply:
Yes, it is. Please check the mailing list. This is my original question: https://lists.freedesktop.org/archives/systemd-devel/2018-January/040122.html and this is a good answer to it: https://lists.freedesktop.org/archives/systemd-devel/2018-January/040124.html

Reply:
I see. Not tested, but it makes sense for me. I did not catch that from your original commit. I hope it is explained as the answer.

Reply:
OK. I will modify my commit. Thanks!